config/configuration.json

```diff
@@ -1079,6 +1079,27 @@
     },
     "minHighwayType": "tertiary"
   },
+  "TallBuildingCheck": {
+    "enabled": true,
+    "bufferDistanceMeters": 1600.0,
+    "minDatasetSizeForStatsComputation": 50.0,
+    "maxLevelTagValue": 100.0,
+    "outlierMultiplier": 18.0,
+    "invalidHeightCharacters": ["~","`","!","@","#","$","%","^","&","*","(", ")","-","_","+","=","[","[","}","]","|","\\",":",";","<",",",">","?","/"],
```

Comment:
Please refer to the naming convention used throughout the file for the config keys.

Reply:
I am not sure I know what you mean by this, please explain when you get the chance!

Reply:
bufferDistanceMeters: -> buffer.distance.meters: ?

Reply:
Yp, the changes look good!